gen/definitions/image_activation.yaml

```diff
@@ -0,0 +1,45 @@
+---
+name: Image Activation
+rest_endpoint: /dna/intent/api/v1/image/activation/device
+no_data_source: true
+res_description:
+  'This resource can activate a software image file on the chosen network device (upgrade the software on the device).
+  Every time this resource is created or re-created, the Catalyst Center considers image activation on the device.
+  (Catalyst Center does not however proceed with activation if the image is already running on the device, assuming
+  the operation has succeeded and noting this fact in its audit log.)
+  <p/>
+  When this resource is destroyed or updated or refreshed, no actions are done either on CatalystCenter or on devices.
+  In effect, this resource currently cannot undo an image activation.'
+no_update: true
+no_delete: true
+no_import: true
+no_read: true
+doc_category: Software Image Management (SWIM)
+skip_minimum_test: true
+attributes:
+  - model_name: deviceUuid
+    type: String
+    data_path: 0
```

Comment:
We would need to add quotes here to prevent it from being interpreted as a number and fail schema validation (https://github.com/CiscoDevNet/terraform-provider-catalystcenter/actions/runs/7846930495/job/21415062464?pr=18).

Reply:
done, thanks